### ISSUE TITLE
Conversion of transliteration from ge'ez

### ISSUE THREAD

**Issue by FrancescaPanini** [SChAth/ScAthiop#116]
The rules on the conversion of the translitteration of ge'ez names (eg Addäbabay -> Āddabābāy?) are unclear: is it possible to clarify how the translitteration needs to be changed from how it appears in the Encyclopaedia in order to avoid inconsistencies and misunderstandings, please?

**Comment by PietroLiuzzo** [comment]
https://github.com/SChAth/ScAthiop/wiki/guidelines#omologation-of-transcription-from-ea

**Comment by abausi** [comment]
Dear Francesca,

please, note that there is a correction to be done in the general rules, 
namely:

ፀ = ḍ (not ś plus dot below)

Moreover: note that there will be inconsistencies at this stage, the 
reason for which is that in previous projects ("Encyclopaedia 
Aethiopica" and "Ethio-SPaRe") were used transcription systems which are 
not good for written texts. For the moment we cannot do anything else, 
unfortunately, but to render them mechanically, since without precise 
linguistic competences and in some cases without checking the texts, we 
cannot decide automatically for all cases.

Ambiguity is still in the rule:


      Omologation of transcription from EA (and Ethio-SPaRe)

  * all the above should be applied to transcriptions from ETHIOPIC only.
  * *ä* for first order should be changed into *a*
  * *a* for fourth order should be changed into *ā*

because (that's the problem!), for "a" with laryngeals we can have both 
first and fourth order.

Note here in details the following ambiguities (with the so-called 
laryngeals):

A, a (at the beginning of a word, A/a vowel occurs only here) > ʾA / ʾĀ, 
ʾa / ʾā: IN PRACTICE: render it with ʾA, ʾa

ʿA (or ʿa) > ʿA / ʿĀ (or ʿa / ʿā): IN PRACTICE: render it with ʿA, ʿa

Ha (or ha) > Ha / Hā(or ha / hā): IN PRACTICE: render it with Ha, ha

Ḥa (or ḥa) > Ḥa / Ḥā (or ḥa / ḥā): IN PRACTICE: render it with Ḥa, ḥa

Ḫa (or ḫa) > Ḫa / Ḫā (or ḫa / ḫā): IN PRACTICE: render it with Ḫa, ḫa

Best,

Alessandro

Il 10.08.2016 21:43, Pietro Liuzzo ha scritto:

**Comment by PietroLiuzzo** [comment]
@abausi may I copy the laryngeals guidances in the guidelines as it is here? I have updated already  ḍ  .

**Comment by FrancescaPanini** [comment]
Thank you very much, this helps putting it into context. 

Just one last confirmation, if possible, please: these rules are to be applied to ALL translitterations from ge'ez, including all place names so that Gondär (Encyclopaedia Aethiopica) will become Gondar? 

Thanks again,
regards
Francesca

**Comment by abausi** [comment]
Please, do it.  If there are further corrections, we will update again.

Il 11.08.2016 09:23, Pietro Liuzzo ha scritto:

**Comment by abausi** [comment]
We should agree on this, since the name has also an Amharic form and I 
do not know how this question was solved. Yet, since we are meanly 
dealing with written sources, I think it should be "Gondar" as well (not 
"Gondär").

Il 11.08.2016 09:50, FrancescaPanini ha scritto:

**Comment by eu-genia** [comment]
> A, a (at the beginning of a word, A/a vowel occurs only here) > ʾA / ʾĀ,
> ʾa / ʾā: IN PRACTICE: render it with ʾA, ʾa
(@abausi)
while this is of course correct, if we always use short a after laryngeals we shall have transcriptions in BM that are different from the ones used in TraCES. E.g., in TraCES አምሃራ will be rendered as ʾAmhāra and in BM as ʾAmhara; ሃይማኖት as Hāymanot and Haymanot, etc. I see a problem of consistency esp. when we link the named entities.

**Comment by abausi** [comment]
This was OBVIOUSLY only the indication for the first step of automatic 
conversion before the actual real transcription is established in each 
case. The "good" final form (either a or ā) can only be verified on the 
single textual passages, and cannot be automatically determined (that's 
why it would have been so important that in cataloguing a different 
system, based upon writing and not pronunciation, was used).

* * *

I am out of office and will have only limited access to email and even less time to answer. Your message can go lost. Probably better to write again later, in September?

Many thanks!

A.B.

* * *

Non sono in servizio e avrò solo un accesso limitato alla posta elettronica, e ancor meno tempo per rispondere. Il messaggio può andar perduto. Forse meglio riscrivere tra un po', verso settembre?

Grazie molte,

A.B.




Il 17.08.2016 13:36, eu-genia ha scritto:

**Comment by abausi** [comment]
Of course, even in the case of the form to be necessarily used for 
reference (apart from variations in single textual passages), no general 
decision can be automatically taken. For linguistically determined 
forms, we can base on the work of TraCES, which considers patterns, 
nominal formation and derivation, morphology etc. but there cases where 
a decision is not easy. This, once again, cannot be automatically done. 
But important is that we start from a uniform situation, where the new 
system is set up (with an a/ā oppoistion, against ä/a of EAe and 
Ethio-SPaRe). The provided indications were intended to start correctly 
the conversion into our (BM and TraCES) system, devised for written 
evidence.

Il 17.08.2016 13:36, eu-genia ha scritto:

**Comment by eu-genia** [comment]
so, @FrancescaPanini and everyone else: we do distinguish between the 1st and the 4th order also after laryngeals when we can/know - so please correct everywhere ʾAmhāra , ʾAzazo Takla Hāymanot, etc.  (@PietroLiuzzo:  to be corrected also in the guidelines)

**Comment by abausi** [comment]
Yes - when we can: but this is a completely different step from the 
question of the automatic conversion from the index of "EAe" (ʾAmhāra is 
probably etymologically "ʾAmḥāra", but difficult to say more).

Il 17.08.2016 14:34, eu-genia ha scritto:

**Comment by eu-genia** [comment]
we do not do any automatic conversion though

**Comment by abausi** [comment]
But this could be considered as a first setp, it would spare time.

Il 17.08.2016 14:49, eu-genia ha scritto:

**Comment by PietroLiuzzo** [comment]
we need then to change in every file not yet manually reworked which comes from EA or Ethio-SPaRe in the following sequence:
- all a to ā 
- all ä to a
- add ʾ at the beginning of each word starting with vowel in the transcription

**Comment by abausi** [comment]
yes, but again: in case of "laryngeal + a", this is just a first 
preliminary step to be later verified, case by case, or to be 
immediately further interpreted, if there is enough evidence for that: 
in particular, it has to be decided whether "a" has to remain or has to 
be further converted into "ā": every complete systematic conversion is 
impossible to apply, because it implies linguistic and other 
considerations, to be seen case by case

Il 17.08.2016 16:57, Pietro Liuzzo ha scritto:

**Comment by eu-genia** [comment]
potentially, all a to ā except if at the beginning or following a laryngeal where they remain a
Ethio-SPaRe files have ʾat the beginning

BUT: the problem remains with the non-Ethiopic words 
what comes out of a work like "Acta Pilati"? or a place like "Abbaa Maalo" (Oromo)? these must not be changed

**Comment by abausi** [comment]
Offline for some days.
      
      A.B.

**Comment by abausi** [comment]
laryngeals here are to be considered only when they precede "a" ("La"), 
not when they follow; in that case we have to do with another 
phenomenon, that does not imply any ambiguity for the transcription: the 
order perfectly corresponds

Il 17.08.2016 17:02, eu-genia ha scritto:

**Comment by abausi** [comment]
OK, sorry, Zhenia had correctly formulated: I misunderstood what she 
wrote ("at the beginning [of a word] or following a laryngeal", i.e. in 
the position "La", is correct)

Il 17.08.2016 18:06, Alessandro Bausi ha scritto:

**Comment by PietroLiuzzo** [comment]
From this conversation and email exchange, the proposed solution is to have a self standing tool (xslt transformation) available for cataloguers. 

in the scenario of some work being carried out on a file from EA or ES, the cataloguer will have to tag with @xml:lang='gez' and @type='transcription' or other all the elements on which to apply the automatic update of the translitteration, then run once and only once the transformation which will operate the substitutions and save them to the same file for further work.

Links importing text from eventually not corrected files will continue to display eventually uncorrected transcriptions. 

As we are not currently working mainly on this records, I will not prioritize this requirement [DAT26](https://github.com/SChAth/ScAthiop/wiki/DAT26) for now.

**Comment by abausi** [comment]
OK so, it is not a priority. Important is that the fiundamental 
coordinates of the process, and the relevant conditions, have been cleared.

Il 17.08.2016 17:33, Pietro Liuzzo ha scritto:
